include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -35,6 +35,7 @@
 #include <deal.II/fe/fe_dgq.h>
 #include <deal.II/fe/fe_poly.h>
 #include <deal.II/fe/fe_q_dg0.h>
+#include <deal.II/fe/fe_raviart_thomas.h>
```

Comment:
Why is this include needed here?

Reply:
Forgot to remove it as I first did some `dynamic_cast` as in `shape_info.templates.h` but figured that I didn't have to do it twice! Thanks